addons/mrp/models/stock_move.py

```diff
@@ -106,7 +106,7 @@ def _compute_product_packaging_qty(self):
             # calculate the number of kit on the move line according to the number of comp
             kit_qty = qty_bom_uom / kit_ratio
             # calculate the quantity needed of packaging
-            move_line.product_packaging_qty = kit_qty / move_line.move_id.product_packaging_id.qty
+            move_line.product_packaging_qty = float_round(kit_qty / move_line.move_id.product_packaging_id.qty, precision_rounding=self.product_uom_id.rounding)
```

Comment:
```suggestion
            move_line.product_packaging_qty = float_round(kit_qty / move_line.move_id.product_packaging_id.qty, precision_rounding=move_line.product_uom_id.rounding)
```
As a rule of thumb, never use `self` from within loop. It's usually a bad idea, and it's one in this case too in case you're computing `product_packaging_qty` on multiple move lines that have different uom at the same time.